travharv/templates/execution_report.ttl

```diff
@@ -0,0 +1,59 @@
+{#
+   This file contains the template for the execution report 
+   :param: execution_report 
+   :param: task_reports
+#}
+
+@prefix prov: <http://www.w3.org/ns/prov#>.
+@prefix sh: <http://www.w3.org/ns/shacl#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix void: <http://rdfs.org/ns/void#>.
+@prefix schema: <http://schema.org/>.
+@prefix travharv: <http://example.org/travharv/ns/>.
+
+<urn:travharv:{{execution_report.id}}> 
+    a prov:Entity, sh:ValidationReport ;
+
+    prov:generatedAtTime "{{execution_report.last_mod}}" ; 
+    travharv:fromContext "{{execution_report.config_name}}" ;
+    sh:result 
+    {%- for t in task_reports %}{% if not loop.last %}{%- for at in t.assertion_reports %} <urn:travharv:assertionresult-{{at.id}}>,{% endfor %}{% endif %}{% if loop.last %}{%- for at in t.assertion_reports %} <urn:travharv:assertionresult-{{at.id}}>{% if not loop.last %},{% endif %}{% endfor %}{% endif %}{% endfor %};
+.
+
+{# Report of a single travharv tast #}
+{%- for t in task_reports %}
+{%- for at in t.assertion_reports %}
```

Comment:
pls apply nesting to `{%%}` sections

the nesting of the generated turtle-triple-text can escape that imho, but with clever `-` here and there we should at least be able to introduce template-structure-nesting for readability